src/emscripten-optimizer/simple_ast.h

```diff
@@ -947,17 +947,18 @@ struct JSPrinter {
     bool integer = wasm::isInteger(d);
 #define BUFFERSIZE 1000
     // f is normal, e is scientific for float, x for integer
-    static char full_storage_f[BUFFERSIZE], full_storage_e[BUFFERSIZE];
+    char full_storage_f[BUFFERSIZE] = {};
+    char full_storage_e[BUFFERSIZE] = {};
```

Comment:
does `= {}` initialize it to 0? I don't think we need that here.

Reply:
Removed.